src/app/components/PageInput/PageInput.module.css

```diff
@@ -1,5 +1,5 @@
 .newPlantName {
```

Comment:
maybe you can combine these classes like:
```js

.newPlantName,
.newTaskTitle,
.newTaskDate{
width: 24.75rem;
  width: 100%;
  height: 3rem;
  box-shadow: 0 0 1.25rem rgba(0, 0, 0, 0.35);
  border-width: 0;
  font-family: var(--font-family);
  font-weight: 300;
  color: var(--primary-text-color);
  font-size: 1.25rem;
  padding-left: 1rem;}
```
  
and just  put the border-radius on `.newTaskTitle` and `.newTaskDate`

to save some code :)